router/batchrouter/asyncdestinationmanager/manager.go

```diff
@@ -43,3 +25,21 @@ func NewManager(destination *backendconfig.DestinationT, backendConfig backendco
 	}
 	return nil, errors.New("invalid destination type")
 }
+
+func NewSFTPManager(destination *backendconfig.DestinationT, backendConfig backendconfig.BackendConfig) (common.AsyncDestinationManager, error) {
```

Comment:
Are we planning to have more SFTP destinations ?

Reply:
Yes, Klaviyo and Salesforce also supports SFTP destinations.